<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QmlJSEditor::Internal::QuickToolBarSettingsPage</class>
 <widget class="QWidget" name="QmlJSEditor::Internal::QuickToolBarSettingsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>433</width>
    <height>428</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Qt Quick Toolbars</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="textEditHelperCheckBoxPin">
        <property name="toolTip">
         <string>If enabled, the toolbar will remain pinned to an absolute position.</string>
        </property>
        <property name="text">
         <string>Pin Qt Quick Toolbar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="textEditHelperCheckBox">
        <property name="text">
         <string>Always show Qt Quick Toolbar</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>207</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
